api/src/glfs.c

```diff
@@ -487,7 +487,7 @@ glfs_free_xlator_options(cmd_args_t *cmd_args)
     xlator_cmdline_option_t *xo = NULL;
     xlator_cmdline_option_t *tmp_xo = NULL;
 
-    if (!&(cmd_args->xlator_options))
+    if (!(&cmd_args->xlator_options))
```

Comment:
I think that this check can be completely removed. `cmd_args->xlator_options` is a `struct list_head` (not a pointer), and taking its address from `cmd_args` will never be `NULL`.

Reply:
Oh yes the check shouldn't be needed.